static/app/stores/sentryAppComponentsStore.tsx

```diff
@@ -34,14 +33,6 @@ const storeConfig: SentryAppComponentsStoreDefinition = {
   getAll() {
     return this.items;
   },
-
-  getComponentByType(type: string | undefined) {
```

Comment:
This was only being used in the HoC, we can just keep that logic there

Reply:
Keeps the store simpler so down the line it's easier to understand how to migrate it off reflux